uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/AbsenceTypeTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_Test_Fabricator_AbsenceType as AbsenceTypeFabricator;
+use CRM_HRLeaveAndAbsences_BAO_AbsenceType as AbsenceType;
+
+/**
+ * Class api_v3_AbsenceTypeTest
+ *
+ * @group headless
+ */
+class api_v3_AbsenceTypeTest extends BaseHeadlessTest {
+
```

Comment:
What about adding a test to assert that an error is thrown when theres no Absence Type with the given ID?